e2e/email-sign-up.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/email-sign-up');
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
```

Comment:
```suggestion
test.beforeEach(async ({ page }) => {
  await page.goto('/email-sign-up');
});
```

We should ideally run the tests in isolation.